GitUI/CommandsDialogs/FormSparseWorkingCopyViewModel.cs

```diff
@@ -162,7 +162,7 @@ public void SaveChanges()
         {
             // Don't abort if !IsWithUnsavedChanges because we have to run IsRefreshWorkingCopyOnSave in either case (e.g. if edited by hand or got outdated)
 
-            // Special case: turning off sparse for a repo � this won't just go smoothly, looks like git still reads the sparse checkout rules, so emptying or deleting them with turning off will just leave you with what you had before
+            // Special case: turning off sparse for a repo � this won't just go smoothly, looks like git still reads the sparse checkout rules, so emptying or deleting them with turning off will just leave you with what you had before
```

Comment:
⚠️ An encoding error occurred and needs to be manually fixed. Don't worry about changing .editorconfig or looking for cases outside this PR (I have a script that can find/fix those separately).